app/views/comments/_comments_list.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="comment-container">
+  <% if logged_in? && !comment.user_already_voted?(current_user) %>
+    <%= render partial: 'votes/comment_upvote', locals: {comment: comment} %>
+    <%= comment.vote_total %>
+    <%= render partial: 'votes/comment_downvote', locals: {comment: comment} %>
+  <% else %>
+    <div><%= submit_tag("Up") %></div>
+    <div><%= comment.vote_total %></div>
+    <div><%= submit_tag("Down") %></div>
+  <% end %>
+</div>
+
+<div class="comment-summary">
+  <p><%=comment.body%></p>
+  <p>posted by <%= link_to comment.commenter.username, user_path(comment.commenter) %></p>
+
+  <ul>
+    <%comment.comments.each do |sub_comment|%>
+      <li><%=sub_comment.body%> posted by <%= link_to sub_comment.commenter.username, user_path(sub_comment.commenter) %></li>
+    <%end%>
+  </ul>
+
+  <div id="new_comment">
+    <% if current_user %>
+    <%= link_to 'add reply', new_comment_comment_path(comment, commentable: {commentable_type: comment.class.to_s, commentable_id: comment.id}) %>
+    <% end %>
+  </div>
+
+  <div id="answer-box">
+    <% if current_user && @question.author == current_user %>
+    <%= form_for comment do |f| %>
+    <%= form_tag(comment_path, method: "patch") %>
```

Comment:
What's going on here - it looks like you're trying to nest forms? That's not legal html